SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -172,7 +172,7 @@ USpatialLatencyTracer* USpatialLatencyTracer::GetTracer(UObject* WorldContextObj
 bool USpatialLatencyTracer::IsValidKey(const TraceKey Key)
 {
 	FScopeLock Lock(&Mutex);
-	return TraceMap.Find(Key);
+	return (TraceMap.Find(Key) != nullptr);
```

Comment:
Brackets not needed, is it better? 
```suggestion
	return TraceMap.Find(Key) != nullptr;
```

Reply:
Not needed, but I find it easier to read. It's not currently in our coding guidelines so feel free to raise.